File: Frontend/src/app/dtos/correlativo-dto.ts
export interface CorrelativoDto {
  idCorrelativo: number;
  correlativoInicial: number;
  correlativoFinal: number;
  correlativoActual: number;
  serie: string;
  fechaCreacion: Date;
  usuario: string;
  idEstado: number;
  estado: string;
}

File: Backend/dtodo-backend/src/main/java/xyz/pangosoft/dtodo/dto/CorrelativoDto.java
package xyz.pangosoft.dtodo.dto;

import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class CorrelativoDto {

	private Long idCorrelativo;
	private Long correlativoInicial;
	private Long correlativoFinal;
	private Long correlativoActual;
	private String serie;
	private LocalDateTime fechaCreacion;
	private String usuario;
	private Integer idEstado;
	private String estado;
}
